### PR TITLE
[CI] Update docker image to node:latest-arm64

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Description**

This PR fixes security vulnerabilities.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@gr455, @cpepper96, assuming this PR is merged (that the changes are good), we're need of having the other adapters similarly updated.

**Comment by leecalcote** [comment]
Track progress and positive impact here - https://artifacthub.io/packages/helm/meshery/meshery?modal=security-report

**Comment by leecalcote** [comment]
@gr455 could be, save for the arch is consistent here: https://github.com/meshery/meshery-kuma/pull/278/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R31

**Comment by gr455** [comment]
> @gr455 could be, save for the arch is consistent here: https://github.com/meshery/meshery-kuma/pull/278/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R31

Ah gotcha, it's still built on amd64. Still, it seems to not work
```
exec /meshery-kuma: no such file or directory
```

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN chmod +x /kubeopenapi-jsonschema
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/nodejs:14
+FROM gcr.io/distroless/nodejs:latest-arm64
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply Dockerfile]: Dropping `arm64`...

**Comment by gr455** [comment]
> Ha! Oh, my goodness. My eyes glossed over the two, reading them as the same thing, when they are dramatically different.

Also either way, the underlying OS is arm and the binary is built for amd64. That may still be a problem right?

**Comment by leecalcote** [comment]
@gr455 thanks for your eagle eyes on this PR. I think with the change to `latest` and only that change, we're good to go. What do you say?

**Comment by gr455** [comment]
> @gr455 thanks for your eagle eyes on this PR. I think with the change to `latest` and only that change, we're good to go. What do you say?

Yep. Looks good to me!

**Pull request merged**